operate/client/src/App/Processes/MigrationView/BottomPanel/index.tsx

```diff
@@ -123,88 +124,72 @@ const BottomPanel: React.FC = observer(() => {
             checkIsRowSelected={handleCheckIsRowSelected(
               selectedSourceFlowNodeIds,
             )}
-            rows={selectableFilteredSourceFlowNodes.map((sourceFlowNode) => {
-              const selectableTargetFlowNodes =
-                processXmlMigrationTargetStore.selectableFlowNodes.filter(
-                  (flowNode) => {
-                    /**
-                     * For boundary events allow only target flow nodes with the same event type
-                     */
-                    if (
-                      hasType({
-                        businessObject: sourceFlowNode,
-                        types: ['bpmn:BoundaryEvent'],
-                      })
-                    ) {
-                      return (
-                        getEventType(sourceFlowNode) === getEventType(flowNode)
-                      );
-                    }
-
-                    /**
-                     * For all other flow nodes allow target flow nodes with the same element type
-                     */
-                    return sourceFlowNode.$type === flowNode.$type;
-                  },
-                );
```

Comment:
This part was moved to `processInstanceMigrationMappingStore` as a computed value.

Reply:
You're still using map on JSX whcih might cause unnecessary rerenders. Doesn't it make sense to compute "all the way" and just map it in the computed value too?

Reply:
I thought about this. In `processInstanceMigrationMappingStore` I would need to add a dependency to `processInstanceMigrationStore`, which I didn't like (because I tend to minimize the dependencies between stores). But maybe it makes sense in this case. Do you have any opinion on this?